static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -298,13 +311,21 @@ class UsageStatsOrganization<
         ),
         score: filtered,
       },
-      dropped: {
-        title: tct('Dropped [dataCategory]', {dataCategory: dataCategoryName}),
+      rateLimited: {
+        title: tct('Rate Limited [dataCategory]', {dataCategory: dataCategoryName}),
+        help: tct(
+          'Rate Limited [dataCategory] were discarded due to rate-limits, abuse, or cardinality limits',
```

Comment:
@jan-auer can you help me with the description here...is it good enough?

Reply:
> due to rate limits or quota.